README.md

```diff
@@ -8,7 +8,8 @@ Variant Normalization relies on some local data caches which you will need to se
 From the _variant_ directory of the repository:
 ```
 pipenv sync
-curl ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/hgnc_complete_set.txt.txt > data/gene_symbols.txt
+curl ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/hgnc_complete_set.txt.txt > data/hgnc_gene_symbols.txt
+curl ftp://ftp.ncbi.nih.gov/refseq/H_sapiens/RefSeqGene/LRG_RefSeqGene > data/refseq_gene_symbols.txt
```

Comment:
if these aren't huge, could we just check + download them if they don't exist like we do w/ the normalizers?

Reply:
Oh you're right. I'll create a new issue.